packages/liveblocks-node/package.json

```diff
@@ -42,7 +43,9 @@
     "@liveblocks/jest-config": "*",
     "@types/node": "^20.7.1",
     "@types/node-fetch": "^2.6.6",
-    "svix": "^0.75.0"
+    "msw": "^2.0.2",
+    "svix": "^0.75.0",
+    "tsd": "^0.29.0"
```

Comment:
No need to add `tsd` as a dependency here. `tsd` is already a global dev dependency for _all_ of our packages, see https://github.com/liveblocks/liveblocks/blob/main/package.json#L24